cluster-operator/src/main/java/io/strimzi/operator/cluster/model/EntityUserOperator.java

```diff
@@ -167,6 +192,10 @@ public static EntityUserOperator fromCrd(Kafka kafkaAssembly) {
                 result.setLogging(userOperatorSpec.getLogging());
                 result.setGcLoggingEnabled(userOperatorSpec.getJvmOptions() == null ? true : userOperatorSpec.getJvmOptions().isGcLoggingEnabled());
                 result.setResources(userOperatorSpec.getResources());
+                if (kafkaAssembly.getSpec().getClientsCa() != null) {
+                    result.setClientsCaValidityDays(kafkaAssembly.getSpec().getClientsCa().getValidityDays());
```

Comment:
What if the the `clientsCa` exists, but the `validityDays and `renewalDays` are not set? You would just overwrite the defaults here with null, or?